interfaces/builtin/upower_observe.go

```diff
@@ -195,7 +195,7 @@ dbus (send)
     bus=system
     path=/org/freedesktop/UPower/devices/**
     interface=org.freedesktop.UPower.Device
-    member=GetHistory
+    member={GetHistory,Refresh}
```

Comment:
There's a weird note about the upower daemon being started in debug mode at https://upower.freedesktop.org/docs/Device.html but looking at the source code this doesn't appear to be the case (anymore?).

Reply:
Yeah I noticed this too - either way, I think it is fine for snapd to allow this and then it's up to upowerd to handle it however it sees fit - but regardless it helps silence a common DENIAL from steam which spams this interface quite often.